app/subscriber/src/TrackPageView.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+class TrackPageView extends React.Component {
```

Comment:
If you're going to create a component, move it to a `/components/snowplow` folder.  All components are organized.